src/eventsub/event.rs

```diff
@@ -446,13 +446,10 @@ fn get_version_event_type_and_message_type_from_text(
     #[derive(Deserialize)]
     struct IEvent {
         subscription: IEventSubscripionInformation,
-        challenge: Option<Empty>,
-        event: Option<Empty>,
+        challenge: Option<serde_json::Value>,
+        event: Option<serde_json::Value>,
```

Comment:
this is a bit wasteful, but it's probably not possible to solve in an easier way.

could maybe do a deserialize impl that just drives the visitor fully, without doing anything. Is there such a thing already?